content/20.pharmaceuticals.md

```diff
@@ -372,6 +372,16 @@ Both doses were well tolerated with no serious events related to the antibody co
 Based on this data, the FDA granted an EUA for REGN-COV2 in patients with mild to moderate COVID-19 who are at risk of developing severe disease [@url:https://www.fda.gov/news-events/press-announcements/coronavirus-covid-19-update-fda-authorizes-monoclonal-antibodies-treatment-covid-19].
 Ongoing efforts are trying to evaluate the efficacy of REGN-COV2 to improve clinical outcomes in hospitalized patients [@clinicaltrials:NCT04426695].
 
+Sotrovimab is the most recent mAb to receive an EUA.
+It was identified in the memory B cells of a 2003 survivor of SARS [@doi:10.1038/nm1080] and was found to be cross-reactive with SARS-CoV-2 [@doi:10.1038/s41586-020-2349-y].
+This cross-reactivity is likely attributable to conservation within the epitope, with 17 out of 22 residues conserved between the two viruses, four conservatively substituted, and one semi-conservatively substituted [@doi:10.1038/s41586-020-2349-y].
+In fact, these residues are highly conserved among sarbecoviruses [@doi:10.1038/s41586-020-2349-y].
```

Comment:
I would add something about how sarbecoviruses include SARS-CoV-2. Not sure this is stated elsewhere in this manuscript.

Reply:
```suggestion
In fact, these residues are highly conserved among sarbecoviruses, a clade that includes SARS-CoV-1 and SARS-CoV-2 [@doi:10.1038/s41586-020-2349-y].
```